### PR TITLE
[#9382] Add tests for GetFeedbackSessionActionTest

### PR DESCRIPTION
**Pull request by ccyccyccy**
Part of #9382

Question: I found some inconsistencies in the behavior of `checkAccessControlForStudentFeedbackSubmission` that affects the test cases. Should I do the change in another PR instead?

### CODE REVIEW

**Comment by madanalogy** [comment]
> Part of #9382
> 
> Question: I found some inconsistencies in the behavior of `checkAccessControlForStudentFeedbackSubmission` that affects the test cases. Should I do the change in another PR instead?

You could submit an issue regarding the inconsistency and then work on it in a separate PR :)

**Comment by ccyccyccy** [comment]
> > Part of #9382
> > Question: I found some inconsistencies in the behavior of `checkAccessControlForStudentFeedbackSubmission` that affects the test cases. Should I do the change in another PR instead?
> 
> You could submit an issue regarding the inconsistency and then work on it in a separate PR :)

Okay! I will continue this PR after the other PR is resolved.

File: src/test/java/teammates/test/cases/webapi/GetFeedbackSessionActionTest.java [hunk 1]
```diff
@@ -113,5 +113,28 @@ protected void testAccessControl() throws Exception {
...
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, fs.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getFeedbackSessionName(),
+                Const.ParamsNames.INTENT, Intent.STUDENT_RESULT.toString(),
+        };
+
+        verifyAccessibleForStudentsOfTheSameCourse(submissionParams);
+        verifyInaccessibleForStudentsOfOtherCourse(submissionParams);
+
+        ______TS("only instructors of the same course can access");
```
Comment (madanalogy) [inline src/test/java/teammates/test/cases/webapi/GetFeedbackSessionActionTest.java:128]: This test case has the same description as one already implemented. Try and think about what this new test case is doing that the other isn't already and rename accordingly.

File: src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java [hunk 2]
```diff
@@ -67,10 +65,6 @@ protected StudentAttributes getStudentOfCourseFromRequest(String courseId) {
      */
     protected void checkAccessControlForStudentFeedbackSubmission(
             StudentAttributes student, FeedbackSessionAttributes feedbackSession) {
-        if (student == null) {
-            throw new EntityNotFoundException(new EntityDoesNotExistException("Student does not exist"));
-        }
-
```
Comment (madanalogy) [inline src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java:73]: This doesn't seem relevant to this PR since you're just adding tests.

Comment (ccyccyccy) [reply src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java:73]: Yup. Will do this in the other PR.

File: src/test/java/teammates/test/cases/webapi/GetFeedbackSessionActionTest.java (hunk 1 above)
Comment (ccyccyccy) [reply src/test/java/teammates/test/cases/webapi/GetFeedbackSessionActionTest.java]: Thanks. Just changed it.

File: src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java (hunk 2 above)
Comment (madanalogy) [reply src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java]: What you need to do is revert these changes to their original, so that if this PR is merged it wouldn't affect what you're doing in your other PR. 

If the new test cases added here however require that these changes take effect, let me know and I'll put this PR on hold until your other PR is resolved :)

Comment (ccyccyccy) [reply src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java]: > What you need to do is revert these changes to their original, so that if this PR is merged it wouldn't affect what you're doing in your other PR.

Ok reverted.

> If the new test cases added here however require that these changes take effect, let me know and I'll put this PR on hold until your other PR is resolved :)

Yup the new test cases added requires those changes. The travis check for this PR would probably fail. Kindly put this PR on hold :)

Comment (madanalogy) [reply src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java]: Can start working on this again when you want to now that the other PR is resolved
